src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -14,11 +14,11 @@ import { ERROR_MESSAGE } from '../../../../shared/constants/api';
 @Component({
   selector: 'alg-item-edit',
   templateUrl: './item-edit.component.html',
-  styleUrls: ['./item-edit.component.scss'],
-  providers: [MessageService]
+  styleUrls: [ './item-edit.component.scss' ],
+  providers: [ MessageService ]
 })
 export class ItemEditComponent implements OnDestroy {
-  itemId : string;
+  itemId : string | null = null;
```

Comment:
Use undefined instead if you have no reason to use null. 
So just `itemId?: string`

Reply:
Fixed